src/export/StructureDefinitionExporter.ts

```diff
@@ -167,6 +168,12 @@ export class StructureDefinitionExporter implements Fishable {
                 rule.sourceInfo
               );
             } else {
+              if (!idRegex.test(invariant.name)) {
```

Comment:
This might be a silly question, but is it possible to use the `validateId` function in common.ts?

Reply:
The catch with that function is that it's a member of the `HasId` class, which means I would need an instance of that or something that mixes it in. `HasId` gets mixed in to a few different FHIR types, but there's no FHIR type to represent an invariant. So I'd need a slightly different mixin (since I'm checking the invariant's `name`, not `id`) for this situation.

Reply:
Ahh gotcha. I somehow managed to miss that it was a part of that class. It seems reasonable that we can't reuse that function and I agree that it doesn't quite make sense to use that class. I was just noticing that we had a very similar error message in a few places, so I was trying to cut down on the repetition we have there, but I can't really think of any other way to reuse that message.